.github/workflows/mass_deployments.yml

```diff
@@ -1,8 +1,7 @@
 name: Mass Deployments
 
 on:
-  schedule:
-    - cron: "0 1 * * *"
+  push
```

Comment:
Why? this shouldn't run on every push!

Reply:
Just updated it to on push to test the workflow and ensure that it's running, once the following run https://github.com/threefoldtech/tfgrid-sdk-ts/actions/runs/6684027673/job/18160843043 is done, i'll revert this change.

Reply:
updated in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1289/commits/c94b27b6be84381880bf490155ba97fdad89cbd3